db/models.py

```diff
@@ -1,5 +1,9 @@
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
 
+from settings import AUTH_USER_MODEL
```

Comment:
The import statement for `AUTH_USER_MODEL` is incorrect. It should be imported from `django.conf` as `from django.conf import settings` and then used as `settings.AUTH_USER_MODEL`.

Reply:
from django.contrib.auth import get_user_model

User = get_user_model()

https://stackoverflow.com/questions/54710952/django-importing-auth-user-model